ansible_anonymizer/anonymizer.py

```diff
@@ -159,11 +155,19 @@ def redact_ip_address(value: str) -> str:
     return str(func(ip))
 
 
+def unquote(value: str) -> str:
+    if value[0] == value[-1] and value[0] in ('"', "'"):
```

Comment:
Maybe it's safer to check `len(value) >= 2`

Reply:
Right!